phoenix-core/src/it/java/org/apache/phoenix/end2end/index/IndexAsyncThresholdIT.java

```diff
@@ -69,51 +70,52 @@
     private final boolean overThreshold;
     private final Mode mode;
 
-    enum Mode{
+    enum Mode {
         NORMAL,
         ASYNC,
         COVERED,
         FUNCTIONAL
-        }
+    }
 
     public IndexAsyncThresholdIT(Long threshold, Long rows, Long columns, Long overThreshold,
                                  Long mode)
-            throws Exception {
+        throws Exception {
         this.tableName = generateUniqueName();
         this.rows = rows;
         this.columns = columns;
         this.overThreshold = overThreshold == 0;
         this.mode = mode.equals(0L) ? Mode.NORMAL :
-                mode.equals(1L) ? Mode.ASYNC :
-                        mode.equals(2L) ? Mode.COVERED :
-                                Mode.FUNCTIONAL;
+            mode.equals(1L) ? Mode.ASYNC :
+                mode.equals(2L) ? Mode.COVERED :
+                    Mode.FUNCTIONAL;
     }
 
     @Parameterized.Parameters
-    public static synchronized Collection<Long[]>  primeNumbers() {
+    public static synchronized Collection<Long[]> primeNumbers() {
         return Arrays.asList(new Long[][]{
-                {100000L, 5000L, 10L, 0L, 0L},
-                {Long.MAX_VALUE, 200L, 100L, 1L, 0L},
-                {0L, 20L, 10L, 1L, 0L},
-                {1L, 20L, 10L, 1L, 1L},
-                {1L, 20L, 10L, 0L, 2L},
-                {1L, 100L, 10L, 0L, 3L},
+            {100000L, 5000L, 10L, 0L, 0L},
```

Comment:
Your autoformatter doesn't seem to be set up right.
Continuation lines should either be matched to the previous expression, or use 8 spaces, not 4.